Marlin/Marlin_main.cpp

```diff
@@ -1691,6 +1691,11 @@ static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0
   feedrate = old_feedrate;
 }
 
+inline void do_blocking_move_to_axis(AxisEnum axis, float where, float feed_rate = 0.0) {
```

Comment:
After this, another audit of the move functions and what calls them, to see what they all actually, ultimately do from each place. Some unrolling might be possible then.